pkg/api/authorizationschema/authorizationschema.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+package immudb.authorizationschema;
+
+import "google/api/annotations.proto";
+import "protoc-gen-swagger/options/annotations.proto";
+
+option go_package = "github.com/codenotary/immudb/pkg/api/authorizationschema";
+option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
+  base_path: "/api/v2",
+  info: {
+    title: "immudb REST API v2";
+    description: "Authorization API"
+  };
+};
+
+message OpenSessionRequestV2 {
+  string username = 1;
+  string password = 2;
+  string database = 3;
+}
+
+message OpenSessionResponseV2 {
+  string token = 1;
+  string serverUUID = 2;
+  int32 expirationTimestamp = 3;
+  int32 inactivityTimestamp = 4;
+
+}
+
+service AuthorizationService {
+  rpc OpenSessionV2(OpenSessionRequestV2) returns (OpenSessionResponseV2) {
```

Comment:
I think it's ok. However splitting authentication with session handling could be analyzed. As users find confusing to open a session against `defaultdb` in order to create a database the very first time... That can be done in another PR is needed

Reply:
On the other hand - I find confusing opening session and then selectin database
maybe we can do this parameter optional, or clearly state this in documentation

Reply:
the idea is to have authentication (without database selection), then you can open a session against a database or create a database without having a session... that will avoid the need of opening a session against defaultdb to be able to create another database